scss/base/_layout.scss

```diff
@@ -5,7 +5,7 @@
   display: none;
 
   @include respond-to('medium') {
-    @include absolute(top 10px left 10px);
+    @include absolute(top 8px left 20px);
```

Comment:
Fair enough!

Although…
```scss
@include absolute(top 8px left 20px);
position: absolute; top: 8px; left: 20px;
```

I think I'd pay for those 4 extra chars to keep code close to the metal.